docs/user_guide/assessment.rst

```diff
@@ -1,21 +1,109 @@
 Assessment
 ==========
 
-Metrics
--------
+Introduction
+------------
 
-.. currentmodule:: fairlearn.metrics
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
+
+The steps of the assesment are as follows:
```

Comment:
The steps of the introduction of the assessment, right?

Reply:
Oh no, this is straight from the tutorial. I think this is supposed to be the steps to conducting an assessment in general